serverless/main.mk

```diff
@@ -31,8 +28,9 @@ ifeq (, $(SLS))
 	$(error "aws cli toolchain is not installed or incorrectly configured.")
 endif
 
-CMD_SLS_SERVICE_INSTALL = $(NPM) install --save-dev
-CMD_SLS_SERVICE_DEPLOY = $(SLS) deploy --env $(ENV) --region $(AWS_REGION) --profile $(AWS_PROFILE) --service $(SVC) -c $(SLS_FILE)
-CMD_SLS_SERVICE_BUILD = cd $(PROJECT_PATH) && make
```

Comment:
Why this has been removed? It will break backwards compatibility with services that require build (golang)

Reply:
fixed